__tests__/app.test.js

```diff
@@ -50,4 +49,34 @@ describe("express app", () => {
         )
     });
   });
+  describe('GET /api/articles/:article_id', () => {
+    it('status: 200 with correct properties', async () => {
+        const { body } = await request(app).get("/api/articles/1")
+        expect(body).toEqual(expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)
+        })
```

Comment:
you can be more specific here, if you are going to articles/1 then you can specifically expect to see article_id : 1, and the same for the rest of the values. check the test data or what you are getting back here to see what the title actually will be for this article, etc.